core/store/src/lib.rs

```diff
@@ -31,37 +31,7 @@ mod db;
 pub mod test_utils;
 mod trie;
 
-pub const COL_BLOCK_MISC: usize = 0;
-pub const COL_BLOCK: usize = 1;
-pub const COL_BLOCK_HEADER: usize = 2;
-pub const COL_BLOCK_INDEX: usize = 3;
-pub const COL_STATE: usize = 4;
-pub const COL_CHUNK_EXTRA: usize = 5;
-pub const COL_TRANSACTION_RESULT: usize = 6;
-pub const COL_OUTGOING_RECEIPTS: usize = 7;
-pub const COL_INCOMING_RECEIPTS: usize = 8;
-pub const COL_PEERS: usize = 9;
-pub const COL_EPOCH_INFO: usize = 10;
-pub const COL_BLOCK_INFO: usize = 11;
-pub const COL_CHUNKS: usize = 12;
-pub const COL_PARTIAL_CHUNKS: usize = 13;
-/// Blocks for which chunks need to be applied after the state is downloaded for a particular epoch
-pub const COL_BLOCKS_TO_CATCHUP: usize = 14;
-/// Blocks for which the state is being downloaded
-pub const COL_STATE_DL_INFOS: usize = 15;
-pub const COL_CHALLENGED_BLOCKS: usize = 16;
-pub const COL_STATE_HEADERS: usize = 17;
-pub const COL_INVALID_CHUNKS: usize = 18;
-pub const COL_BLOCK_EXTRA: usize = 19;
-/// Store hash of a block per each height, to detect double signs.
-pub const COL_BLOCK_PER_HEIGHT: usize = 20;
-pub const COL_LAST_APPROVALS_PER_ACCOUNT: usize = 21;
-pub const COL_MY_LAST_APPROVALS_PER_CHAIN: usize = 22;
-pub const COL_STATE_PARTS: usize = 23;
-pub const COL_EPOCH_START: usize = 24;
-// Map account_id to announce_account
-pub const COL_ACCOUNT_ANNOUNCEMENTS: usize = 25;
-const NUM_COLS: usize = 26;
+pub use db::DBCol::{self, *};
```

Comment:
I would remove `* ` here and do `use near_store::DBCol::*;` in `store.rs`.

Reply:
Hmm there isn't a store.rs in core/store